libindy_vdr/src/ffi/mod.rs

```diff
@@ -72,6 +76,29 @@ pub extern "C" fn indy_vdr_set_cache_directory(path: FfiStr) -> ErrorCode {
     }
 }
 
+#[no_mangle]
+pub extern "C" fn indy_vdr_set_ledger_txn_cache(capacity: usize, expire_offset: u64) -> ErrorCode {
+    catch_err! {
+        debug!("Setting pool ledger transactions cache: capacity={}, expire_offset={}", capacity, expire_offset);
+        *write_lock!(LEDGER_CACHE_STRATEGY)? = Some(Arc::new(CacheStrategyTTL::new(capacity, expire_offset.into(), None, None)));
+        Ok(ErrorCode::Success)
+    }
+}
+
+#[no_mangle]
+pub extern "C" fn indy_vdr_set_ledger_txn_fs_cache(
+    capacity: usize,
+    expire_offset: u64,
+    path: FfiStr,
+) -> ErrorCode {
```

Comment:
Could we combine these two methods and just make the path an optional parameter, using the in-memory cache if it's empty or null?

Reply:
done